lib/tasks/test_vmdb.rake

```diff
@@ -3,7 +3,7 @@ require_relative "./evm_test_helper"
 if defined?(RSpec) && defined?(RSpec::Core::RakeTask)
 namespace :test do
   namespace :vmdb do
-    task :setup => :setup_db
+    task :setup => [:verify_no_db_access_loading_rails_environment, :setup_db]
```

Comment:
I have no idea if the other code will react ok to Rails env being loaded by a prerequisite task.